package.json

```diff
@@ -57,6 +57,7 @@
     "redux-thunk": "2.2.0",
     "requirejs": "2.3.5",
     "rtlcss": "2.2.1",
+    "sass": "^1.49.10",
```

Comment:
@aht007 Can you help me understand why you removed node-sass and installed sass as part of this upgrade?

Reply:
`node-sass` has been deprecated for a long time now and we faced several issues with it when moving towards node16. Also `dart-sass`(sass) is the recommended replacement for it which works nicely with node16 and hence the change.
Reference Docs: 

- https://2u-internal.atlassian.net/wiki/spaces/AC/pages/2626116/Node+16+Upgrade (Internal, Attaching Screenshot

<img width="845" alt="Screenshot 2023-01-18 at 11 58 23 PM" src="https://user-images.githubusercontent.com/60315450/213270237-056cde84-de7e-46e1-b578-6678d2f0db50.png">
)

- https://github.com/sass/node-sass#readme

Reply:
Got it, thank you!

For context, the edx-platform build pipeline still uses libsass-python (which is deprecated as well). I am currently looking into replacing with, either with node-sass or dart-sass. There's discussion in #wg-developer-experience if you're curious.